data/json/hobbies.json

```diff
@@ -91,5 +91,208 @@
     "description": "You enjoyed camping in the vast wilderness as a means of escaping the pace of modern life.  The wilderness isn't filled with people, so it must be safe.  Right?  It must be.",
     "points": 2,
     "skills": [ { "level": 2, "name": "survival" }, { "level": 2, "name": "firstaid" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "hobby_pilot",
+    "name": "Helicopter Pilot",
+    "description": "You got your pilot's license, and earned a living ferrying businessmen and tourists around.",
+    "points": 2,
+    "skills": [ { "level": 2, "name": "driving" }, { "level": 2, "name": "mechanics" } ],
+    "proficiencies": [ "prof_helicopter_pilot" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "reading",
+    "name": "Reading",
+    "description": "You love reading, you just never seemed to have enough time.  Maybe the cataclysm is a blessing in disguise.",
+    "points": 1,
+    "traits": [ "LOVES_BOOKS" ],
+    "skills": [ { "level": 2, "name": "chemistry" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "chem_cooking",
+    "name": "Chemical Cooking",
+    "description": "You decided to cook your fix instead of waiting for your next one.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "chemistry" }, { "level": 1, "name": "cooking" } ]
```

Comment:
If we go this route I think we should take the option to select the trait away. I'd say for now it shouldn't add any skills, until we are able to add theory XP.

Reply:
I think this is meant to go on the 'loves books' above.

Reply:
I think I'm misunderstanding something? This first batch of hobbies was based around each skill so there's some diversity. Is it skills in hobbies, or just chemical cooking in particular? Also, I'm not sure what you mean by select the trait away since there's no traits for this particular hobby

Reply:
Oh, okay that makes a lot more sense lol

Reply:
yes this was for "love books", consequence of walking and typing on phone. What I meant to say for this one was just "we should add proficiencies here too."